tools/flowc/backends/common.flow

```diff
@@ -659,8 +660,9 @@ getFcBackendConfigs(config : CompilerConfig) {
 				htmlAdditionalScripts,
 				htmlPredefinedParams,
 				getConfigParameterDef(config.config, "working-dir", ""),
-				if (htmlManifest == "1") "manifest.json" else htmlManifest)
-			) else None();
+				if (htmlManifest == "1") "manifest.json" else htmlManifest,
+				getConfigParameterDef(config.config, "html-translate", "") == "1"
```

Comment:
Maybe it is better to name it "html-no-translate" to avoid confusion?

Reply:
Option is renamed to 'html-no-translate', default value is 1.